data_collection/web/guides.rst

```diff
@@ -9,4 +9,224 @@ Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
+```
+
+In one of following link schemas:
+
+ - file extension is at the very end of a link eg. `http://example.com/file.7z` or `http://example.com/article?click=file.7z`
+ - file extension directly preceeds query part (`?`), eg. `http://example.com/article/file.7z?source=user#how-to`
+ - file extension directly preceeds fragment part (`#`) `http://example.com/article?target=file.7z#how-to`
+ - file extension is at the end of query param, eg. `http://example.com/article?click=file.7z&page=3`
+
+Customizing list of file extensions
+"""""""""""""""""""""""""""""""""""
+You can customize list of file extansions you want to track as downloads. For example, if you want to track only images as downloads, you can use `setDownloadExtensions` function to replace the list this:
+
+```
+// track clicks on images only
+_paq.push(['setDownloadExtensions', "png|jpg|webp|gif"]);
+```
+
+You can add new extensions, to an existing list with `addDownloadExtensions`:
+
+```
+// add other image formats
+_paq.push(['setDownloadExtensions', "svg|xcf"]);
+```
+
+Or remove some of extenstions from the existing list with `removeDownloadExtensions`:
+
+```
+_paq.push(['removeDownloadExtensions', "jpg|jpeg"]);
+_paq.push(['trackPageView']);
+```
+
+Tracking downloads beyond file extensions
+"""""""""""""""""""""""""""""""""""""""""
+
+If your case of download links does not fall in above cases you still have options to use, to tell tracker that link should be tracked as a download.
+
+You can add a download attribute to a link HTML tag. eg. 
+
+```<a href="/target-file" download>```
+
+Or if you have to be strict with your HTML, you can add a HTML tag class. Default classes are `piwik_download` and `piwik-download`. Eg. 
+
+```<a href="/taget-file" class="piwik-download">```
+
+Additionally you can define your custom CSS classes for download links with ours :ref:`Javascript Tracker API<force-tracking-download-using-css-class>`. Eg. 
+
+```
+_paq.push(['setDownloadClasses', "custom-download-class"]);
+_paq.push(['trackPageView']);
+```
+
+or you can define a list of classes at once, by passing an array list of CSS classes:
+
+```
+_paq.push(['setDownloadClasses', ["custom-download-class", "other-download-class", "another-class"]]);
+_paq.push(['trackPageView']);
+```
+
+and in HTML code:
+
+```<a href="/taget-file" class="custom-download-class">```
+
+.. note::
+You have to remember that using `setDownloadClasses` always overwrite current list of CSS classes.
+
+
+Tracking downloads with inline Javascript
+"""""""""""""""""""""""""""""""""""""""""
+
+There is another alternative for above methods. You can track a download with inline javascript. Insert inline javascript to HTML tag with onclick attribute:
+
+```
+<a href="https://piwik.pro/document-url" target="_blank" onClick="_paq.push(['trackLink', 'https://piwik.pro/document-url', 'download']);">Download document</a>
+```
+
+Tracking downloads when using log importer
```

Comment:
Log importer is not part of 16.0, please hide/remove it. I think better place for this information is Web log analytics doc

Reply:
I did hide it with ```.. hide::```

Reply:
:nice: